### DISCUSSION TITLE
Answer one by one

### DISCUSSION THREAD

**Discussion by FrontEnd07** [def-studio/telegraph#451] [Q&A]
There is a method in the documentation DISPATCH But I don’t understand how to use it, how can I respond to a message one by one after the command? And how can you get the contents of a message by identifier?

**Comment by fabio-ivona** [comment]
`dispacth()` is used to send a message to telegram through laravel's queues

you can use `send()` if you don't need/use queues

**Reply by FrontEnd07** [reply]
I need to use queues like:
Your name?
Николай 
How old are you?
45
And at the end answer, your name is Nikolai, you are 45 years old

**Reply by fabio-ivona** [reply]
in this case you should build a conversation logic by yourself

**Reply by GrOoOmChIk20** [reply]
Can you write a piece of code on how to do this?

**Reply by fabio-ivona** [reply]
It is not a trivial task, it requires you to keep track of the conversation status (could use telegraph storage system or db/cache)

On a message received you should check for the conversation status for that particular user/chat and handle it accordingly
